app/helpers/container_group_helper/textual_summary.rb

```diff
@@ -122,4 +122,26 @@ def textual_lives_on
       )
     }
   end
+
+  def textual_containers_statuses_summary
+    %i(waiting running terminated)
+  end
+
+  def containers_statuses_summary
```

Comment:
I think that this could be enough:

    def containers_statuses_summary
      @containers_statuses_summary ||= @record.containers_states_summary
    end